README.md

```diff
@@ -67,12 +67,6 @@ Finally, launch JupyterLab with
 jupyter lab
 ```
 
-To update the existing environment, run
-
-```console
-conda env update -f environment.yml
```

Comment:
Maybe instead of deleting it, you could update it to:
```bash
conda env update -f binder/environment.yml
```

Reply:
It's already there:
https://github.com/empa-scientific-it/python-tutorial/blob/f7149925eddcdbcd875a42c6a7a1c47b35e9a1a3/README.md?plain=1#L56-L60

Reply:
Incidentally: Conda here is barely useful. All the packages are installed with `pip`, and using Conda slows down quite a lot the building of the Docker image. If you try it locally, you'll see why.

Reply:
But as the smart people say: "if it ain't broken, don't fix it".